PluginCore/ScintillaNet/Lexers.cs

```diff
@@ -80,9 +80,13 @@ public enum CPP
         TRIPLEVERBATIM = 21,
         HASHQUOTEDSTRING = 22,
         PREPROCESSORCOMMENT = 23,
-        WORD3 = 24,
-        WORD4 = 25,
-        WORD5 = 26,
+        PREPROCESSORCOMMENTDOC = 24,
+        USERLITERAL = 25,
+        TASKMARKER = 26,
+        ESCAPESEQUENCE = 27,
```

Comment:
This could be interesting to get something like this:

![](http://i.imgur.com/sQybXsU.png)

I don't think the set of escape sequences can be customized though, so things that aren't escape sequences in Haxe [but C++](http://en.cppreference.com/w/cpp/language/escape) would get highlighted.